requirements.txt

```diff
@@ -1,6 +1,5 @@
 rustworkx>=0.13.0
 numpy>=1.17,<2
-ply>=3.10
```

Comment:
Very pleased to see this going, especially since it's not been maintained as an actual Python package since 2020.

Reply:
This was actually my motivation for this PR, I was looking at the requirements list and forgot this was in there.